src/frontend/screens/Library/index.tsx

```diff
@@ -82,6 +82,24 @@ export default function Library(): JSX.Element {
   const { t } = useTranslation()
   const backToTopElement = useRef(null)
 
+  //Remember scroll position
+  useEffect(() => {
```

Comment:
let's use `useLayoutEffect` here instead to avoid the `jump` from the top to the current scroll

Reply:
Ah! Cheers didn't know that hook existed :)